docs/cli/get-inventory-information.md

```diff
@@ -0,0 +1,159 @@
+# Gest Inventory Information
+
+ANTA CLI provides a set of entrypoints to get data from your local inventory.
+
+## Get all configured tags
+
+Since most commands in anta support tags filtering, this command helps you list all available tags configured in your inventory.
+
+### Command overview
+
+```bash
+anta get tags --help
+Usage: anta get tags [OPTIONS]
+
+  Get list of configured tags in user inventory.
+
+Options:
+  --log-level, --log [debug|info|warning|critical]
+                                  Logging level of the command
+  --help                          Show this message and exit.
+```
+
+### Example
+
+Let consider the following inventory:
+
+```yaml
+anta_inventory:
+  hosts:
+  - host: 192.168.0.10
+    name: spine01
+    tags: ['fabric', 'spine']
+  - host: 192.168.0.11
+    name: spine02
+    tags: ['fabric', 'spine']
+  - host: 192.168.0.12
+    name: leaf01
+    tags: ['fabric', 'leaf']
+  - host: 192.168.0.13
+    name: leaf02
+    tags: ['fabric', 'leaf']
+  - host: 192.168.0.14
+    name: leaf03
+    tags: ['fabric', 'leaf']
+  - host: 192.168.0.15
+    name: leaf04
+    tags: ['fabric', 'leaf']
+```
+
+To get list of all configured tags in your CLI, run the following command:
+
+```bash
+$ anta get tags
+Tags found:
+[
+  "all",
+  "fabric",
+  "leaf",
+  "spine"
+]
+None
+
+* note that tag all has been added by anta
```

Comment:
```suggestion
* note that tag 'all' has been added by anta
```

Reply:
Maybe better with oneline code ?

```suggestion
* note that tag `all` has been added by anta
```

Reply:
could be - is this output part of the command or is it something added? (I am not sure as it is in the code block)

Reply:
Correct, CLI output is not using quote but color instead which is not available in this context.